onnxruntime/python/onnxruntime_pybind_mlvalue.h

```diff
@@ -17,6 +17,8 @@
 #include "core/framework/ort_value.h"
 #include "core/session/inference_session.h"
 
+PYBIND11_MAKE_OPAQUE(std::vector<OrtValue>);
```

Comment:
Maybe surround with `if (NOT onnxruntime_MINIMAL_BUILD)`

Reply:
Do you mean this job Linux CPU Minimal Build E2E CI Pipeline is not covering this?

Reply:
I am not sure whether minimal build wants dlpack support. @edgchen1 might know better?

Reply:
I'm not too familiar with this. Is there a good reason to include dlpack support in a minimal build?
If not, perhaps exclude it for now.

Reply:
dlpack is a protocol to exchange tensors accross libraries without copying the data, only the structure describing it. numpy and torch are implementing it and the latest release. It is interesting for big tensors or in an environment where memory is constraint but in that case, i assume users would prefer the C++ API not the python one.

Reply:
I believe using a minimal build with the Python API is uncommon. @skottmckay please correct me if I'm wrong. If this is only enabled when building the Python API that may be ok.